recipes/nvcc/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+# Default to using `nvcc` to specify `CUDA_HOME`.
+if [ -z ${CUDA_HOME+x} ]
+then
+    CUDA_HOME="$(dirname $(dirname $(which nvcc)))"
+fi
+
+# Set `CUDA_HOME` in an activation script.
```

Comment:
If `CUDA_HOME` is already set should we store the value into `BACKUP_CUDA_HOME` or similar and restore it in the deactivate script?

Reply:
We could do that. Though this will probably be a little gross to write/read (we need to escape some stuff). Also it probably won't matter that much if users are already using this with our Docker image (the intent here).